{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:52:03.057Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:52:03.128Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:52:03.128Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:52:03.129Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:52:03.130Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:52:03.130Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:52:03.131Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:52:03.131Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:52:03.140Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:52:03.140Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:52:03.179Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:52:03.180Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:52:03.207Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:52:06.758Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:52:13.282Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:52:19.901Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:52:26.488Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:52:33.069Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:52:39.722Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:52:46.376Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:52:53.037Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:52:59.710Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:53:06.036Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:53:06.097Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:53:06.098Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:53:06.098Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:53:06.099Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:53:06.100Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:53:06.100Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:53:06.101Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:53:06.108Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:53:06.109Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:53:06.143Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:53:06.143Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:53:06.169Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:53:09.774Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:53:16.492Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:53:23.135Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:53:29.775Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:53:36.506Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:53:43.127Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:53:49.728Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:53:56.364Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:54:03.007Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:54:08.829Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:54:08.899Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:54:08.900Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:54:08.901Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:54:08.902Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:54:08.902Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:54:08.903Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:54:08.903Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:54:08.911Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:54:08.911Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:54:08.943Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:54:08.943Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:54:08.978Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:54:12.521Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:54:19.089Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:54:25.678Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:54:32.300Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:54:38.885Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:54:45.483Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:54:52.073Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:54:58.692Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:55:05.292Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:55:09.984Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:55:10.026Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:55:10.026Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:55:10.027Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:55:10.027Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:55:10.028Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:55:10.029Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:55:10.029Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:55:10.036Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:55:10.037Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:55:10.098Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:55:10.098Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:55:10.154Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:55:13.673Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["Error: getaddrinfo ENOTFOUND quote-postgres\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"],"timestamp":"2025-03-19T14:55:20.276Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:55:25.506Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:55:25.544Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:55:25.544Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:55:25.544Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:55:25.545Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:55:25.545Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:55:25.546Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:55:25.546Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:55:25.552Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:55:25.552Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:55:25.573Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:55:25.573Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:55:25.597Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:55:25.601Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:55:28.615Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:55:31.635Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:55:34.650Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:55:37.708Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:55:40.731Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:55:43.746Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:55:46.831Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:55:49.849Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:55:53.596Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:55:53.636Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:55:53.637Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:55:53.637Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:55:53.638Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:55:53.639Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:55:53.640Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:55:53.640Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:55:53.646Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:55:53.647Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:55:53.683Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:55:53.683Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:55:53.719Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:55:53.725Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:55:56.747Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:55:59.790Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:02.807Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:05.823Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:08.849Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:11.868Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:14.892Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:17.918Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:56:19.803Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:56:19.836Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:56:19.836Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:56:19.836Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:56:19.837Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:56:19.837Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:56:19.838Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:56:19.838Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:56:19.843Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:56:19.843Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:56:19.862Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:56:19.863Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:56:19.889Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:19.895Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:22.917Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:25.948Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:28.965Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:31.985Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:35.005Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:38.017Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:41.041Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:44.062Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:56:46.183Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:56:46.219Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:56:46.220Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:56:46.220Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:56:46.221Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:56:46.221Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:56:46.221Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:56:46.222Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:56:46.228Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:56:46.228Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:56:46.250Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:56:46.251Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:46.282Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:56:46.286Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:49.303Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:52.322Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:55.353Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:56:58.372Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:01.386Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:04.409Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:07.427Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:10.448Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:57:12.594Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:57:12.629Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:57:12.629Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:57:12.630Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:57:12.630Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:57:12.631Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:57:12.631Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:57:12.631Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:57:12.636Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:57:12.637Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:57:12.658Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:57:12.659Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:12.689Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:57:12.693Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:15.717Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:18.736Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:21.765Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:24.790Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:27.821Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:30.844Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:33.869Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:36.890Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:57:38.964Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:57:38.996Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:57:38.996Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:57:38.997Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:57:38.997Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:57:38.998Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:57:38.998Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:57:38.998Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:57:39.004Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:57:39.004Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:57:39.051Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:57:39.052Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:39.082Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:57:39.086Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:42.100Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:45.117Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:48.134Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:51.158Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:54.185Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:57:57.207Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:00.237Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:03.259Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:58:05.179Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:58:05.208Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:58:05.208Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:58:05.209Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:58:05.209Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:58:05.210Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:58:05.210Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:58:05.210Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:58:05.214Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:58:05.215Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:58:05.235Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:58:05.235Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:05.260Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:58:05.264Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:08.282Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:11.301Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:14.318Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:17.345Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:20.367Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:23.389Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:26.417Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:29.442Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:58:31.374Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:58:31.405Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:58:31.405Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:58:31.406Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:58:31.406Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:58:31.406Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:58:31.407Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:58:31.407Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:58:31.412Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:58:31.412Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:58:31.430Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:58:31.431Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:31.458Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:58:31.461Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:34.482Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:37.506Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:40.520Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:43.541Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:46.566Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:49.582Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:52.597Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:55.614Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:58:57.663Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:58:57.697Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:58:57.697Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:58:57.698Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:58:57.698Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:58:57.699Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:58:57.699Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:58:57.699Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:58:57.704Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:58:57.705Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:58:57.725Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:58:57.726Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:58:57.753Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:58:57.757Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:00.779Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:03.796Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:06.811Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:09.831Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:12.848Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:15.863Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:18.880Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:21.898Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:59:23.965Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:59:24.001Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:59:24.001Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:59:24.002Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:59:24.002Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:59:24.003Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:59:24.003Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:59:24.003Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:59:24.009Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:59:24.010Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:59:24.033Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:59:24.034Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:24.060Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:59:24.064Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:27.080Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:30.115Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T14:59:44.103Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T14:59:44.139Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T14:59:44.139Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T14:59:44.139Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T14:59:44.140Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T14:59:44.140Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:59:44.141Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T14:59:44.141Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T14:59:44.146Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T14:59:44.147Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T14:59:44.170Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T14:59:44.170Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:44.201Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T14:59:44.205Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:47.224Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:50.249Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:53.263Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:56.293Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T14:59:59.310Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:02.323Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:05.339Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:08.362Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T15:00:10.437Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T15:00:10.474Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T15:00:10.474Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T15:00:10.474Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T15:00:10.475Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T15:00:10.475Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T15:00:10.476Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T15:00:10.476Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T15:00:10.481Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T15:00:10.481Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T15:00:10.502Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T15:00:10.502Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:10.530Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T15:00:10.534Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:13.555Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:16.578Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:19.604Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:22.624Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:25.638Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:28.670Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:31.686Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:34.705Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T15:00:36.812Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T15:00:36.850Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T15:00:36.851Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T15:00:36.851Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T15:00:36.852Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T15:00:36.853Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T15:00:36.854Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T15:00:36.854Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T15:00:36.861Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T15:00:36.862Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T15:00:36.885Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T15:00:36.885Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:36.919Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T15:00:36.923Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:39.936Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:42.955Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:45.985Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:49.004Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:52.039Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:55.059Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:00:58.071Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:01.091Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T15:01:03.182Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T15:01:03.215Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T15:01:03.215Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T15:01:03.216Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T15:01:03.216Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T15:01:03.217Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T15:01:03.217Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T15:01:03.217Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T15:01:03.222Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T15:01:03.222Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T15:01:03.242Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T15:01:03.243Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T15:01:03.262Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:03.273Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:06.291Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:09.313Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:12.327Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:15.339Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:18.362Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:21.382Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:24.400Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:27.438Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T15:01:29.385Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T15:01:29.422Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T15:01:29.422Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T15:01:29.422Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T15:01:29.423Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T15:01:29.423Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T15:01:29.423Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T15:01:29.424Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T15:01:29.428Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T15:01:29.429Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T15:01:29.451Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T15:01:29.451Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:29.480Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T15:01:29.483Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (2)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:32.527Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (3)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:35.560Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (4)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:38.587Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (5)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:41.607Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (6)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:44.640Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (7)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:47.657Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (8)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:50.681Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (9)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:53.714Z"}
{"context":"NestFactory","level":"info","message":"Starting Nest application...","timestamp":"2025-03-19T15:01:55.646Z"}
{"context":"InstanceLoader","level":"info","message":"AppModule dependencies initialized","timestamp":"2025-03-19T15:01:55.679Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T15:01:55.679Z"}
{"context":"InstanceLoader","level":"info","message":"PassportModule dependencies initialized","timestamp":"2025-03-19T15:01:55.680Z"}
{"context":"InstanceLoader","level":"info","message":"JwtModule dependencies initialized","timestamp":"2025-03-19T15:01:55.680Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigHostModule dependencies initialized","timestamp":"2025-03-19T15:01:55.681Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T15:01:55.681Z"}
{"context":"InstanceLoader","level":"info","message":"ConfigModule dependencies initialized","timestamp":"2025-03-19T15:01:55.681Z"}
{"context":"InstanceLoader","level":"info","message":"MulterModule dependencies initialized","timestamp":"2025-03-19T15:01:55.687Z"}
{"context":"InstanceLoader","level":"info","message":"MailerModule dependencies initialized","timestamp":"2025-03-19T15:01:55.687Z"}
{"context":"InstanceLoader","level":"info","message":"HomeModule dependencies initialized","timestamp":"2025-03-19T15:01:55.707Z"}
{"context":"InstanceLoader","level":"info","message":"MailModule dependencies initialized","timestamp":"2025-03-19T15:01:55.708Z"}
{"context":"InstanceLoader","level":"info","message":"I18nModule dependencies initialized","timestamp":"2025-03-19T15:01:55.725Z"}
{"context":"TypeOrmModule","level":"error","message":"Unable to connect to the database. Retrying (1)...","stack":["error: database \"quote\" does not exist\n    at Parser.parseErrorMessage (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"],"timestamp":"2025-03-19T15:01:55.734Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmCoreModule dependencies initialized","timestamp":"2025-03-19T15:01:58.896Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T15:01:58.897Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T15:01:58.897Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T15:01:58.897Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T15:01:58.897Z"}
{"context":"InstanceLoader","level":"info","message":"TypeOrmModule dependencies initialized","timestamp":"2025-03-19T15:01:58.897Z"}
{"context":"InstanceLoader","level":"info","message":"ForgotModule dependencies initialized","timestamp":"2025-03-19T15:01:58.898Z"}
{"context":"InstanceLoader","level":"info","message":"SessionModule dependencies initialized","timestamp":"2025-03-19T15:01:58.898Z"}
{"context":"InstanceLoader","level":"info","message":"AuthFacebookModule dependencies initialized","timestamp":"2025-03-19T15:01:58.899Z"}
{"context":"InstanceLoader","level":"info","message":"AuthGoogleModule dependencies initialized","timestamp":"2025-03-19T15:01:58.899Z"}
{"context":"InstanceLoader","level":"info","message":"AuthTwitterModule dependencies initialized","timestamp":"2025-03-19T15:01:58.899Z"}
{"context":"InstanceLoader","level":"info","message":"AuthAppleModule dependencies initialized","timestamp":"2025-03-19T15:01:58.899Z"}
{"context":"InstanceLoader","level":"info","message":"ArticlesModule dependencies initialized","timestamp":"2025-03-19T15:01:58.900Z"}
{"context":"InstanceLoader","level":"info","message":"UsersModule dependencies initialized","timestamp":"2025-03-19T15:01:58.900Z"}
{"context":"InstanceLoader","level":"info","message":"FilesModule dependencies initialized","timestamp":"2025-03-19T15:01:58.900Z"}
{"context":"InstanceLoader","level":"info","message":"AuthModule dependencies initialized","timestamp":"2025-03-19T15:01:58.900Z"}
{"context":"RoutesResolver","level":"info","message":"UsersController {/api/users} (version: 1):","timestamp":"2025-03-19T15:01:58.923Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/users, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.924Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/users, GET} (version: 1) route","timestamp":"2025-03-19T15:01:58.925Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/users/:id, GET} (version: 1) route","timestamp":"2025-03-19T15:01:58.925Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/users/:id, PATCH} (version: 1) route","timestamp":"2025-03-19T15:01:58.925Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/users/:id, DELETE} (version: 1) route","timestamp":"2025-03-19T15:01:58.926Z"}
{"context":"RoutesResolver","level":"info","message":"FilesController {/api/files} (version: 1):","timestamp":"2025-03-19T15:01:58.926Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/files/upload, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.926Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/files/:path, GET} (version: 1) route","timestamp":"2025-03-19T15:01:58.926Z"}
{"context":"RoutesResolver","level":"info","message":"AuthController {/api/auth} (version: 1):","timestamp":"2025-03-19T15:01:58.927Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/email/login, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.928Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/email/register, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.928Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/email/confirm, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.928Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/forgot/password, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.929Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/reset/password, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.929Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/me, GET} (version: 1) route","timestamp":"2025-03-19T15:01:58.929Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/refresh, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.929Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/logout, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.929Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/me, PATCH} (version: 1) route","timestamp":"2025-03-19T15:01:58.929Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/me, DELETE} (version: 1) route","timestamp":"2025-03-19T15:01:58.929Z"}
{"context":"RoutesResolver","level":"info","message":"AuthFacebookController {/api/auth/facebook} (version: 1):","timestamp":"2025-03-19T15:01:58.930Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/facebook/login, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.930Z"}
{"context":"RoutesResolver","level":"info","message":"AuthGoogleController {/api/auth/google} (version: 1):","timestamp":"2025-03-19T15:01:58.930Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/google/login, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.930Z"}
{"context":"RoutesResolver","level":"info","message":"AuthTwitterController {/api/auth/twitter} (version: 1):","timestamp":"2025-03-19T15:01:58.930Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/twitter/login, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.930Z"}
{"context":"RoutesResolver","level":"info","message":"AuthAppleController {/api/auth/apple} (version: 1):","timestamp":"2025-03-19T15:01:58.930Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/auth/apple/login, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.931Z"}
{"context":"RoutesResolver","level":"info","message":"HomeController {/api}:","timestamp":"2025-03-19T15:01:58.931Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/, GET} route","timestamp":"2025-03-19T15:01:58.931Z"}
{"context":"RoutesResolver","level":"info","message":"ArticlesController {/api/articles} (version: 1):","timestamp":"2025-03-19T15:01:58.931Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/articles, POST} (version: 1) route","timestamp":"2025-03-19T15:01:58.931Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/articles, GET} (version: 1) route","timestamp":"2025-03-19T15:01:58.931Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/articles/:id, GET} (version: 1) route","timestamp":"2025-03-19T15:01:58.932Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/articles/:id, PATCH} (version: 1) route","timestamp":"2025-03-19T15:01:58.932Z"}
{"context":"RouterExplorer","level":"info","message":"Mapped {/api/articles/:id, DELETE} (version: 1) route","timestamp":"2025-03-19T15:01:58.932Z"}
{"context":"NestApplication","level":"info","message":"Nest application successfully started","timestamp":"2025-03-19T15:01:58.938Z"}
